dask_yarn/cli.py

```diff
@@ -440,42 +442,41 @@ async def run():
     ),
 )
 def worker(nthreads=None, memory_limit=None):  # pragma: nocover
-    enable_proctitle_on_current()
-    enable_proctitle_on_children()
-
     if memory_limit is None:
         memory_limit = int(skein.properties.container_resources.memory * 2 ** 20)
     if nthreads is None:
         nthreads = skein.properties.container_resources.vcores
 
+    worker_options = dict(memory_limit=memory_limit, nthreads=nthreads)
     app_client = skein.ApplicationClient.from_current()
 
     scheduler = app_client.kv.wait("dask.scheduler").decode()
-
-    loop = IOLoop.current()
-
-    worker = Nanny(
+    worker_class = os.environ.get("worker_class", "dask.distributed.Nanny")
```

Comment:
This is already a default config option. Do we want to double up on the default value here?

Is there a reason why you aren't using the Dask config system?

Reply:
This is injected into the worker running in a yarn container so there won't be a dask config setup.